build-system/server/app.js

```diff
@@ -47,9 +47,33 @@ const {
 } = require('./recaptcha-router');
 const {getServeMode} = require('./app-utils');
 const {logWithoutTimestamp} = require('../common/logging');
+const {log} = require('../common/logging');
+const {red} = require('kleur/colors');
 const {renderShadowViewer} = require('./shadow-viewer');
 const {replaceUrls, isRtvMode} = require('./app-utils');
 
+/**
+ * Respond with content received from a URL when SERVE_MODE is "cdn".
+ * @param {express.Response} res
+ * @param {string} cdnUrl
+ * @return {Promise<boolean>}
+ */
+async function passthroughServeModeCdn(res, cdnUrl) {
+  if (SERVE_MODE !== 'cdn') {
+    return false;
+  }
+  try {
+    const response = await fetch(cdnUrl);
+    res.status(response.status);
+    res.send(await response.text());
```

Comment:
Does this resolve in time for `res.send()` ??

Reply:
Yep! This translates to:
```
response.text().then(text => res.send(text))
```